docs/Azure_IoT_Hub_integration/Device_operations/Upgrading_firmware.md

```diff
@@ -0,0 +1,117 @@
+    - **Method timeout** - specify a timeout for direct method result notification.
+
+5. Once you have provided the required data, click **Invoke method**.
+
+    ![Direct Method invoke button](images/dir_method_invoke.png "Direct Method invoke button")
+
+6. After a short moment, you should be able to see the direct method result in the **Result** field.     
+
+    ![FOTA direct method result](images/direct_method_result.png "Direct Method result")
+
+    - The `200` as the "status" parameter value means that the firmware upgrade task was completed successfully.
+    - Importantly, the result "payload" value will be needed for other FOTA actions like status check or cancellation, so be sure to copy it to your clipboard if needed.
+
+!!! tip
+    Out of all the parameters provided in Firmware upgrade direct method payload, only two are mandatory:
+
+      - **name** - the unique file name used for firmware identification.
+      - **firmwareUrl** - the URL used by Coiote DM to download the firmware file and include it as a resource.   
+
+    Therefore it is correct to include only those two in the payload, as in here:
+      ```
+      {
+      "name": "anjay-firmware",
+      "firmwareUrl": "https://repository.avsystem.com/artifactory/gitlfs/demo.fw-pkg",
+      }
+      ```
+
+## Checking Firmware upgrade status
+
+To check the status of a scheduled firmware upgrade, follow these steps:
+
+1. In the **Direct Method** tab of your device, provide data for the following fields:
+    - **Method Name** - paste the `checkFirmwareUpdateStatus` direct method name here.
+    - **Payload** - use the payload displayed in the Firmware upgrade result field (remember to replace the placeholder value with your copied value):
+    ```
+    {
+    "fotaId": "fotaIdReturnedByScheduleOperation"
+    }
+    ```
+2. Click **Invoke method**.
+
+    ![Direct Method invoke button](images/dir_method_invoke.png "Direct Method invoke button")
+
+3. Check the direct method status in the Result field:
+
+    ![FOTA status check result](images/direct_method_status_check.png "Direct Method result - status")
+
+## Cancelling the Firmware upgrade procedure
+
+To cancel the firmware upgrade procedure, follow these steps:
+
+1. In the **Direct Method** tab of your device, provide data for the following fields:
+    - **Method Name** - paste the `cancelFirmwareUpdate` direct method name here.
+    - **Payload** - use the payload displayed in the Firmware upgrade result field (remember to replace the placeholder value with your copied value):
+      ```
+      {
+      "fotaId": "fotaIdReturnedByScheduleOperation"
+      }
+      ```
+2. Click **Invoke method**.
+
+    ![Direct Method invoke button](images/dir_method_invoke.png "Direct Method invoke button")
+
+3. Check the direct method status in the Result field:
+
+    ![FOTA Direct method cancellation](images/direct_method_cancel.png "Direct Method result - cancel")
```

Comment:
Please add some summary at the end. Maybe try it on your own or something, because it looks like something is missing here.

Reply:
not sure if a summary is a great idea here - I went for a 'See also' section at the end.